libs/ledger-live-common/src/families/celo/api/hubble.ts

```diff
@@ -89,5 +137,40 @@ export const getAccountDetails = async (address: string, accountId: string) => {
     balance,
     spendableBalance,
     operations,
+    lockedBalance,
+    nonvotingLockedBalance,
   };
 };
+
+export const getValidatorGroups = async (): Promise<CeloValidatorGroup[]> => {
+  const validatorGroups = await fetchValidatorGroups();
+
+  const result = validatorGroups.map((validatorGroup) => ({
+    //TODO: toLowerCase()?
```

Comment:
I'm considering using always lowercased validatorgroup addresses since indexer and node return them in different formats